lib/helpers/getNParents.js

```diff
@@ -0,0 +1,18 @@
+import { getDomNodeProfile } from './getDomNodeProfile';
```

Comment:
Circular dependency here. It doesn't look like we use this method anywhere outside `getDomNodeProfile`, why not just keep it in there?

Reply:
re: circular dependency - whoops good catch! I feel like I fixed it by cheating a bit by moving the basicProfile code to its own file/method, but it gets the job done. open to objections tho

re: why not just keep it there - originally I had it there, but the recording lib has several additional functions that do more heavy lifting, so the file and its spec started to feel cluttered. I noticed that the `getLinkParent` function was already living in its own file (not created by us) and felt that moving things out as such would make the file clearer and the specs a bit more easily and thoroughly targeted.

Reply:
Just to second this:   Splitting stuff up into basic profile is very helpful in the recording library. I anticipate over time introducing more and more functionality from the recording library into the tracking library and think this will be a useful way to group the output.